src/coreclr/nativeaot/Runtime/eventtrace.cpp

```diff
@@ -282,6 +302,9 @@ void EtwCallback(
     if (context->RegistrationHandle == Microsoft_Windows_DotNETRuntimeHandle) {
         providerIndex = DotNETRuntime;
         providerContext = MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_DOTNET_Context;
+    } else if (context->RegistrationHandle == Microsoft_Windows_DotNETRuntimeRundownHandle) {
+        providerIndex = DotNETRuntimeRundown;
+        providerContext = MICROSOFT_WINDOWS_DOTNETRUNTIME_RUNDOWN_PROVIDER_DOTNET_Context;
```

Comment:
Does this callback also need to check for `EndRundownKeyword` to trigger a rundown for ETW scenarios?